source/slang/slang-parser.cpp

```diff
@@ -2243,6 +2243,20 @@ namespace Slang
         return parseThisTypeExpr(parser);
     }
 
+    static Expr* parseTupleTypeExpr(Parser* parser)
+    {
+        parser->ReadToken(TokenType::LParent);
```

Comment:
I am worried that this is going to introduce more ambiguity. I suggest we remove the parenthesis syntax for now, and just parse it as a Tuple<x,y> syntax.